server/src/main/scala/za/co/absa/atum/server/api/exception/DatabaseError.scala

```diff
@@ -23,5 +23,6 @@ object DatabaseError {
   case class GeneralDatabaseError(message: String) extends DatabaseError
   case class ConflictDatabaseError(message: String) extends DatabaseError
   case class NotFoundDatabaseError(message: String) extends DatabaseError
+  case class ErrorInDataDatabaseError(message: String) extends DatabaseError
```

Comment:
**Non-consequential for this PR:**
What if `DatabaseError` would have `status`, and `statusText` fields?

Reply:
GeneralDatabaseError is used to represent all kinds of exceptions (not only fa-db exceptions); therefore we don't have any status and status test.

Reply:
Good point.